seed/views/v3/properties.py

```diff
@@ -1758,6 +1758,63 @@ def delete_inventory_document(self, request, pk):
         doc_file.delete()
         return JsonResponse({"status": "success"})
 
+    @swagger_auto_schema(
+        manual_parameters=[
+            AutoSchemaHelper.query_org_id_field(),
+            AutoSchemaHelper.query_integer_field(
+                name="access_leevl_instance_id", required=True, description="Access Level Instance to move properties to"
+            ),
+        ],
+        request_body=AutoSchemaHelper.schema_factory(
+            {"property_view_ids": ["integer"]},
+            required=["property_view_ids"],
+            description="A list of property view ids to move between alis",
+        ),
+    )
+    @api_endpoint_class
+    @ajax_request_class
+    @action(detail=False, methods=["POST"])
+    @has_perm_class("can_modify_data")
+    def move_properties_to(self, request):
+        """
+        Move properties to a different ali
+        """
+        org_id = self.get_organization(request)
+        ids = request.data.get("property_view_ids", [])
+        ali_id = request.query_params.get("access_level_instance_id", None)
+
+        if not ali_id:
+            return JsonResponse({"status": "error", "message": "Target ALI not found"}, status=status.HTTP_404_NOT_FOUND)
+
+        # filter ids based on request user's ali
+        ali = AccessLevelInstance.objects.get(pk=request.access_level_instance_id)
+        target_ali = AccessLevelInstance.objects.filter(pk=ali_id, lft__gte=ali.lft).first()
```

Comment:
I'm a little confused by this line, can you explain?

Reply:
The intention here is to ensure that the user has access to the ALI they are targeting with the move.  the ALI on line 1790 is the user's assigned access level instance, and the ali_id is the intended target ALI's pk.  The filter attempts to load the target ali by comparing their left sides.